### ISSUE TITLE
Crash on Lost Spawn

### ISSUE THREAD

**Issue by CaptainPetRock** [long-war-2/lwotc#1287]
[Saves.zip](https://github.com/long-war-2/lwotc/files/6289950/Saves.zip)

Crash on Lost Spawn with a mission with the Lost Sitrep, Cleared Configs/Validated/Launched with only the required mods and still crashes.

I have included three saves.
Gamer = Right before I first encountered the issue
Gamer2= Save right before the mission
Test= Me running the mission again after reloading, still has issue

**Comment by pledbrook** [comment]
This seems to be connected with Lost Brutes, but not everyone is affected. No idea why. Note that if you throw a grenade on XCOM's turn to force the Lost to spawn on that turn (rather than at the end of it), the crash doesn't occur.

**Comment by pledbrook** [comment]
So I tried Gamer2 as well (the Geoscape save) and ran the mission. Didn't crash for me, even when a Brute spawned in after the end of XCOM's turn. The tactical save (Gamer) did crash, though. Very odd.

**Comment by pupujuku73** [comment]
I'm getting the same problem.  I'm still early in my game (mid-May 2035) so I have not encountered Brutes yet.  Lost on covert-op ambush missions so far have been working as normal.  The game freezes then crashes when The Lost are being spawned now; both after my turn, and during when I use a grenade.

**Comment by pledbrook** [comment]
@pupujuku73 Which version of LWOTC are you playing?

**Comment by pupujuku73** [comment]
I was playing dev26 at the time.; I didn't realize BETA4 was just out.  When I updated, and reloaded the mission, it worked ok... I saw the Lost Brute.  Cute!  Just as a side note:  I kept playing the same game before updating and got another mission with "The Lost" sitrep... it worked fine, but there were no Brutes that spawned.  That would seem to further suggest that Brutes were causing the issue earlier.
